### PR TITLE
[excel] (Range, Worksheet) Adding information about preview find methods

### PR DESCRIPTION
**Pull request by AlexJerabek**
This adds sections about the upcoming string searching methods of Range and Worksheet. It also cleans up the spacing in the Worksheet article.

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c93a2db](https://github.com/OfficeDev/office-js-docs-pr/pull/652/commits/c93a2db6434a15865345fecc8011f42c515d10aa)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c93a2db](https://github.com/OfficeDev/office-js-docs-pr/pull/652/commits/c93a2db6434a15865345fecc8011f42c515d10aa)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c93a2db](https://github.com/OfficeDev/office-js-docs-pr/pull/652/commits/c93a2db6434a15865345fecc8011f42c515d10aa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Find/docs/excel/excel-add-ins-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Find/docs/excel/excel-add-ins-worksheets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812282039379995-652%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kLu0mYdTlHneyKSwHTAg9%2FBAUl3qwhtQk4RYdNNqIlU%3D&st=2018-12-28T20%3A35%3A52Z&se=2019-01-28T20%3A40%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-worksheets.md [hunk 1]
```diff
@@ -254,52 +253,78 @@ Excel.run(function (context) {
...
+
     return context.sync()
         .then(function() {
             console.log(`The value of the cell in row 2, column 5 is "${cell.values[0][0]}" and the address of that cell is "${cell.address}"`);
         })
 }).catch(errorHandlerFunction);
 ```
 
-## Get a range within a worksheet
+### Find all ranges with matching text (preview)
```
Comment (kbrandl) [inline docs/excel/excel-add-ins-worksheets.md]: Suggest renaming this section -> **Find data in a worksheet (preview)** and promoting it to an H2. Reason being -- "find data in a worksheet" describes the task that the reader is likely to have in mind -- they aren't likely to think of it in terms of "find all ranges with matching text" (even though that technically describes what the API does).

`## Find data in a worksheet (preview)`

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c41c801](https://github.com/OfficeDev/office-js-docs-pr/pull/652/commits/c41c80142470d4dbd867af7f0660d9d6272eddfb)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c41c801](https://github.com/OfficeDev/office-js-docs-pr/pull/652/commits/c41c80142470d4dbd867af7f0660d9d6272eddfb)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

**Comment by AlexJerabek** [comment]
Thanks @kbrandl. I've addressed your feedback and incorporated the notes from the related snippet checkins. I also added a small section about the Range.find behavior pertaining to single cell ranges.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c41c801](https://github.com/OfficeDev/office-js-docs-pr/pull/652/commits/c41c80142470d4dbd867af7f0660d9d6272eddfb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Find/docs/excel/excel-add-ins-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Find/docs/excel/excel-add-ins-worksheets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812290007441170-652%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NLutSvrGC3VoYu%2FxGFtHHrtgNoh8QKWK7ufDqSOy3hU%3D&st=2018-12-29T00%3A04%3A02Z&se=2019-01-29T00%3A09%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**